src/components/accordion/index.stories.mdx

```diff
@@ -0,0 +1,129 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import AccordionPanel from './AccordionPanel';
+import AccordionItem from './AccordionItem';
+import AccordionButton from './AccordionButton';
+
+import ListItem from '../list/ListItem';
+import List from '../list';
+import { CheckmarkIcon, TooltipIcon } from '../icons';
+
+import Accordion from './index';
+
+<Meta title="Components/Disclosure/Accordion" component={Accordion} />
```

Comment:
This component should be in another path. You can verify it in figma file as well.


```suggestion
<Meta title="Patterns/Navigation/Accordion" component={Accordion} />
```

Reply:
hmm, okay moved according to figma, but in my opinion it's not only navigation component, and will be usable in different places

Reply:
Would you mind doble checking with the Designer?. We need to be in sync. Thanks!